test/03-error-handling.js

```diff
@@ -1,24 +0,0 @@
-const tap = require('tap')
```

Comment:
very, very unlikely for npm packages... i think those cases (e.g. bower and other package managers) should be handled in another module called `get-license-bower` or something similiar...